File: app/src/main/java/com/example/worldnewsin/SimpleVideoList.java
package com.example.worldnewsin;

import java.io.Serializable;
import java.util.List;

public class SimpleVideoList implements Serializable {
    public List<SimpleVideo> items;

    public List<SimpleVideo> getItems() {
        return items;
    }

    public int getSize() {
        return items == null ? 0 : items.size();
    }

    public SimpleVideo getVideo (int position) {
        return items.get(position);
    }
}

File: app/src/main/java/online/litterae/worldnewsin/model/SimpleVideoList.kt
package online.litterae.worldnewsin.model

import online.litterae.worldnewsin.model.SimpleVideo
import java.io.Serializable

class SimpleVideoList : Serializable {
    var items: List<SimpleVideo> = ArrayList<SimpleVideo>()

    val size: Int
        get() = items.size

    fun getVideo(position: Int): SimpleVideo {
        return items[position]
    }
}